recipes-sota/aktualizr/aktualizr_git.bb

```diff
@@ -5,13 +5,17 @@ SECTION = "base"
 LICENSE = "MPL-2.0"
 LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=9741c346eef56131163e13b9db1241b3"
 DEPENDS = "boost curl openssl jansson libsodium ostree"
-SRCREV = "4e9344ae375a444f02b964dca52fe808010d17df"
-PV = "1.0+git${SRCPV}"
+RDEPENDS_${PN} = "python-petname"
```

Comment:
I'm not super-keen on bringing in a whole python dependency just to make up a random name--what about implementing this in Aktualizr directly?